modules/nf-core/unzip/main.nf

```diff
@@ -34,4 +33,19 @@ process UNZIP {
         7za: \$(echo \$(7za --help) | sed 's/.*p7zip Version //; s/(.*//')
     END_VERSIONS
     """
+
+    stub:
+    def args = task.ext.args ?: ''
+    if ( archive instanceof List && archive.name.size > 1 ) { error "[UNZIP] error: 7za only accepts a single archive as input. Please check module input." }
+    prefix = task.ext.prefix ?: ( meta.id ? "${meta.id}" : archive.baseName)
+    """
+    mkdir "${prefix}"
+    touch "${prefix}"/ncbi.map
+    touch "${prefix}"/ncbi.tre
```

Comment:
this is only true for the old test data... not sure what a good stub test would be, because anything can be the output... maybe just creating the `prefix` directory is enough?

Reply:
checking that there is actually something in the extracted directory would be the cherry on the top. but it's good as it is at the moment if you want to move on 🙂